### PR TITLE
fix(env): update required node engine to v18.18

### PR DESCRIPTION
**Pull request by OnkarRuikar**
- porting from yari PR https://github.com/mdn/yari/pull/11373

### CODE REVIEW

File: .nvmrc [hunk 1]
```diff
@@ -1 +1 @@
-v18
+v18.18
```
Comment (OnkarRuikar) [inline .nvmrc:1]:
```suggestion
v18
```

**Review by argl — APPROVED** [review]

**Comment by yin1999** [comment]
Hey @OnkarRuikar. I think the problem metioned in mdn/yari#11373 would not affect this repo. I've made a test with `node v18.0.0`, and I found that the min version required by dependencies in this repo could be `18.12.0`:

image

And I can install the dependencies when the node version set to `v18.12.0`:

image

---

Should we set it to `>=18.12.0` instead?

File: package.json [hunk 2]
```diff
@@ -24,7 +24,7 @@
     "prepare": "husky || true"
   },
   "engines": {
-    "node": ">=18.0.0"
+    "node": ">=18.18.0"
```
Comment (OnkarRuikar) [inline package.json:27]: Unlike Content, this repo doesn't have a dependency on Yari. So we can use the minimum required `18.12.0`:
```suggestion
    "node": ">=18.12.0"
```

**Review by yin1999 — APPROVED** [review]
Thank you @OnkarRuikar

**Pull request merged**